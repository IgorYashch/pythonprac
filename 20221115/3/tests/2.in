alp = Alpha(k=11, d='lsfdj', l=None, c=10, z=2, a=42)
alp.e = 123
try:
    alp.foo = 39748
except AttributeError:
    print("Hell, no!!!")

print(alp)


alq = AlphaQ(k=11, d='lsfdj', l=None, c=10, z=2, a=42)
alq.e = 123
try:
    alq.foo = 39748
except AttributeError:
    print("Hell, no!!!")
    
print(alq)
