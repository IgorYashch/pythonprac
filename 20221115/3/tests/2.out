Hell, no!!!
a: 42, c: 10, d: lsfdj, e: 123, k: 11, l: None, z: 2
Hell, no!!!
a: 42, c: 10, d: lsfdj, e: 123, k: 11, l: None, z: 2
